Rakefile

```diff
@@ -1,13 +1,30 @@
+# frozen_string_literal: true
+
+require 'rubocop/rake_task'
+RuboCop::RakeTask.new(:rubocop) do |task|
+  # These make the rubocop experience maybe slightly less terrible
+  task.options = ['-D', '-S', '-E']
```

Comment:
Are there long options for these to better understand what they are?

Reply:
updated it